src/server/test/crossSite.js

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* This file tests ???. */
+
+// Run in OED Docker web container terminal/shell: npm run testsome src/server/test/crossSite.js
+
+const { chai, mocha, expect, app, testUser } = require('./common');
+
+mocha.describe('Cross site', () => {
+    mocha.it('test 1', async () => {
+        const res = await chai.request(app).post('/api/csv/readings')
+            .field('email', testUser.username)
+            .field('password', testUser.password)
+            .field('meterName', "&lt;img&#32;src&#47;onerror&#61;alert&#40;document&#46;domain&#41;&gt;")
+            .field('timeSort', "increasing")
```

Comment:
Not required but I think I mentioned that a number of the fields could be removed to simplify the test.